src/main/java/pl/javastart/foodieapp/order/OrderController.java

```diff
@@ -37,23 +33,23 @@ public String addItemToOrder(@RequestParam Long itemId, Model model) {
     }
 
     @GetMapping("/zamowienie")
```

Comment:
nazwy po angielsku to raz a dwa raczej czasownik w sensie nazwe akcji trzeba podac jaka chcesz

Reply:
zrobione